packages/xs-extra/xapi-cli-protocol.master/descr

```diff
@@ -0,0 +1,4 @@
+The xapi toolstack daemon which implements the XenAPI
```

Comment:
Minor point but I think `that` instead of `which` would be better. `That` would introduce an essential restriction whereas `which` adds an (optional) qualification.

Reply:
The descriptions are all wrong (but xapi's) actually... We should probably discuss what to put in all of them